package.json

```diff
@@ -58,6 +58,7 @@
     "watch": "gulp watch",
     "prettify": "prettier --write \"{lib,tests,automation,typings}/**/*.js\" --config ./node_modules/resin-lint/config/.prettierrc",
     "lint": "resin-lint lib/ tests/ && resin-lint --typescript --fix automation/ lib/ typings/ tests/",
+    "update": "npx ts-node@^8.0.0 --transpile-only ./automation/update-module.ts",
```

Comment:
Is `npx ts-node@^8.0.0` necessary? The `devDependencies` section already requires `ts-node@^8.6.2`, so maybe it's enough to have:
```
"update": "ts-node --transpile-only ./automation/update-module.ts",
```

Reply:
Also: other `ts-node` entries above include `-P automation/tsconfig.json`